src/dataflow/src/render/reduce.rs

```diff
@@ -1198,6 +1198,8 @@ where
 
 /// Accumulates values for the various types of accumulable aggregations.
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+// Numeric is 100 bytes
+#[allow(variant_size_differences)]
```

Comment:
this is perhaps a stupid question, but what does enabling this do? I tried searching for it with no luck

Reply:
oh i see now, this is just a lint, and we are explicitly allowing the lint. I'm curious why we didn't try out the heap allocation for numerics?